compiler/src/iree/compiler/Codegen/Dialect/GPU/IR/IREEGPUAttrs.cpp

```diff
@@ -832,9 +832,8 @@ MMAScheduleAttr::getContractionLayout(vector::ContractionOp contractOp) const {
     llvm::errs() << "Getting mma layouts for:\n" << contractOp << "\n";
     llvm::errs() << "For schedule: " << *this << "\n";
   });
-  if (opInfo.getOpKind() == VectorContractOpInfo::OpKind::UNKNOWN) {
-    LLVM_DEBUG({ llvm::errs() << "Unknown contraction kind\n"; });
-    return failure();
+  if (opInfo.getKDims().size() != 1) {
+    return contractOp->emitError("NYI: > 1 k dims");
```

Comment:
What is 'NYI'? I found 'New York Islanders' :P

Reply:
I'd change it to `"Unimplemented: > 1 k dims"`

Reply:
fair enough :) done!